src/component/page/tiltaksoversikt/bruker-oversikt/TabellBody.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import { Rad } from './Rad';
 import { Bruker } from '../../../../api/data/bruker';
+import { BrukerSortering } from './TabellHeader';
 
 interface TabellBodyProps {
+	sortering?: BrukerSortering;
```

Comment:
Er det en spesiell grunn til at disse sendes nedover komponenttreet isteden for å hooke Radkomponenten på store?

Reply:
Personlig så foretrekker jeg å enten ha "dumme" eller "smarte" komponenter som trekker inn ting utenifra som f.eks global state. I dette tilfellet så er tabellen smart, mens de små komponentene inni er dumme. Det kan fort bli grisete å trekke inn global state i komponenter som ligger veldig dypt

Reply:
Ja, når det ikke går på bekostning av noe annet så er jeg enig i det. Det er generelt viktigere å ikke re-rendre komponenter som som ikke trenger det hver gang verdien endrer seg enn å ha flere dumme komponenter og syns ikke det er grisete i det hele tatt. Når dataene overføres via komponent som ikke har behov for dataene så syns jeg også det er mer diskuterbart.

Men er litt enig i at det er logisk å gjøre det sånn i det her tilfellet siden det er to children av parenten til Rad som trenger sortering og alle komponentene under må kanskje re-rendres når sorteringen endrer seg uansett(?).